dapps/digital-twin-detail/src/components/DigitalTwinDetail.vue

```diff
@@ -58,40 +58,107 @@
                         icon="mdi mdi-dots-vertical"
                       />
                     </template>
-                    <b-dropdown-item href="#">TODO Clone</b-dropdown-item>
+                    <b-dropdown-item
+                      @click="duplicateTwin()">
+                      {{ '_twin-detail.data.context-menu.duplicate-twin' | translate }}
+                    </b-dropdown-item>
                     <b-dropdown-item
                       @click="exportTwinTemplate()"
                     >{{ '_twin-detail.data.context-menu.export-template' | translate }}</b-dropdown-item>
                   </b-dropdown>
 
-                  <evan-modal ref="exportModal">
-                    <template v-slot:header>
-                      <h5 class="modal-title">
-                        {{ `_twin-detail.data.context-menu.export-template` | translate }}
-                      </h5>
-                    </template>
-
-                    <template v-slot:body>
-                      <evan-loading v-if="exporting" />
+                  <evan-swipe-panel
```

Comment:
shall we outsource the swipe-panels to own components?

Reply:
Moved the swipe panels to a seperated `TwinInteractions` component.